bcda/cclf/cclf.go

```diff
@@ -417,11 +419,11 @@ func orderACOs(cclfMap map[string]map[metadataKey][]*cclfFileMetadata) []string
 
 func getPriorityACOs(db *sql.DB) []string {
 	const query = `
-	SELECT trim(both '["]' from g.x_data::json->>'cms_ids') "aco_id" 
-	FROM systems s JOIN groups g ON s.group_id=g.group_id 
-	WHERE s.deleted_at IS NULL AND g.group_id IN (SELECT group_id FROM groups WHERE x_data LIKE '%A%' and x_data NOT LIKE '%A999%') AND
-	s.id IN (SELECT system_id FROM secrets WHERE deleted_at IS NULL);
-	`
+    SELECT trim(both '["]' from g.x_data::json->>'cms_ids') "aco_id" 
```

Comment:
Not sure why this is happening. Used gofmt.  And it doesn't look like this in vscode.

Reply:
Looks good! I believe we're entering the tabs vs. spaces debate.

This was tabs, now it's spaces. Since it's in the multi-line string, `gofmt` doesn't seem to care. So I think we're good!